pkg/config/dynamic/middlewares.go

```diff
@@ -237,14 +237,17 @@ func (h *Headers) HasSecureHeadersDefined() bool {
 
 // IPStrategy holds the ip strategy configuration.
 type IPStrategy struct {
-	Depth       int      `json:"depth,omitempty" toml:"depth,omitempty" yaml:"depth,omitempty" export:"true"`
-	ExcludedIPs []string `json:"excludedIPs,omitempty" toml:"excludedIPs,omitempty" yaml:"excludedIPs,omitempty"`
+	Depth              int      `json:"depth,omitempty" toml:"depth,omitempty" yaml:"depth,omitempty" export:"true"`
+	ExcludedIPs        []string `json:"excludedIPs,omitempty" toml:"excludedIPs,omitempty" yaml:"excludedIPs,omitempty"`
+	TrueClientIPHeader string   `json:"trueClientIPHeader" toml:"trueClientIPHeader" yaml:"trueClientIPHeader"`
```

Comment:
using `true` here seems contrary to the strategy deployed, which is `CustomHeaderStrategy`, in addition to considering the assumptions one makes with regards to booleans. Should we consider renaming this to `CustomClientIPHeader`?

Reply:
@notsureifkevin: I'm fine with changing the naming, however the reason I named it this is true client IP is the terminology for getting the actual IP address for the requesting client. I'd be open to getting more feedback on this because I'm fine either way as long as it makes sense :)

Reply:
@notsureifkevin doesn't look like we are getting much other feedback on this, at least so far... do you think I should go ahead and make that change?